charts/operator-wandb/charts/parquet/values.yaml

```
@@ -17,6 +18,7 @@ extraCors: []
 
 common: {}
 deployment: {}
+cronjob: {}
```

```suggestion
cronJob: {}
```
Also please update it in the template